ignite/engine/engine.py

```diff
@@ -300,7 +300,7 @@ def has_event_handler(self, handler: Callable, event_name: Optional[Any] = None)
                 to ``None`` to search all events.
         """
         if event_name is not None:
-            self._assert_non_filtered_event(event_name)
+            # self._assert_non_filtered_event(event_name)
```

Comment:
We can remove commented code. Have you checked the correct behaviour of `has_event_handler` on filtered events ?

Reply:
It seems correct. However I noticed that if a handler is registered with a filtered event, you can remove it using non filtered event (Ser test_custom_event.py)